TasksApp.java

```diff
@@ -1 +1,71 @@
+package com.company;
 
+import java.util.Objects;
+import java.util.function.DoubleUnaryOperator;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.IntStream;
+
+public class TasksApp {
+
+    public static void main(String[] args) {
+        //Task 1
+        System.out.println("Task 1: ");
+        Predicate<Object> condition = Objects::isNull;
+        Function<Object, Integer> ifTrue = obj -> 0;
+        Function<CharSequence, Integer> ifFalse = CharSequence::length;
+        Function<String, Integer> safeStringLength = ternaryOperator(condition, ifTrue, ifFalse);
+        System.out.println(safeStringLength.apply("Hello world!")); //12
+        System.out.println(safeStringLength.apply("")); //0
+        //Task 2
+        System.out.println("Task 2: ");
+        pseudoRandomStream(13).mapToObj(x -> x + " ").forEach(System.out::print);
+        //Task 3
+        System.out.println("\nTask 3: ");
+        System.out.println(integrate(x -> 1, 0, 10)); //10.0
+        System.out.println(integrate(x -> x + 2, 0, 10)); //70.0
+        System.out.println(integrate(x -> Math.sin(x) / x, 1, 5)); //0.603848174577285
+    }
+
+    private static <T, U> Function<T, U> ternaryOperator(
+            Predicate<? super T> condition,
+            Function<? super T, ? extends U> ifTrue,
+            Function<? super T, ? extends U> ifFalse) {
+        return string -> condition.test(string) ? ifTrue.apply(string) : ifFalse.apply(string);
+    }
+
+    public static IntStream pseudoRandomStream(int seed) {
+        IntStream stream = IntStream.iterate(seed, n -> mid(n * n)).limit(8);
```

Comment:
```suggestion
        return IntStream.iterate(seed, n -> ((n * n) / 10 % 1000));
```
limit is unnecessary here